CONTRIBUTING.md

```diff
@@ -0,0 +1,58 @@
+# Contributing
+
+## When to submit an ADR
+
+The purpose of an ADR is to record an over-arching technical decision made by the development team in the Archivematica 
+project. An ADR is akin to a policy in other contexts- it describes a way that future features or changes should be 
+implemented, rather than describing the features themselves (to submit a feature idea, head to the Issues repo).  
+Developers will use ADRs to understand how code should be written and analysts and end-users to understand implementation 
+decisions.
+
+Examples of situations when an ADR is appropriate:
+
+* A change to how code is formatted 
+* Major architectural changes (e.g. bringing together or separation of components, upgrade of major dependencies, etc.)
+* Deprecation of existing features
+
+ADRs may be written retroactively to make a record of core features or functionality.
+
+## How to submit an ADR
+
+ADRs must be submitted as a pull request to this repo. To get started, first review currently accepted ADRs, then use the 
+template to write yours.
```

Comment:
We should link to the template: [here](https://github.com/artefactual-labs/archivematica-architectural-decisions/blob/master/template.md)